process/datadog_checks/process/process.py

```diff
@@ -503,3 +504,9 @@ def _filter_by_user(self, user, pids):
                 pass
 
         return filtered_pids
+
+    @staticmethod
+    def _is_win32():
+        is_64bits = sys.maxsize > 2 ** 32
```

Comment:
Note that this will tell you if python is 32bits, not if windows is 32bits. I guess we always build python appropriately on windows?

Reply:
Yup, but I dont know if the part that is supossed to not work on win32 does not work because of windows 32 or because of python32 🤷‍♀️